common/source/docs/common-remoteid.rst

```diff
@@ -20,6 +20,7 @@ MAVLink enabled devices that connect to the autopilot:
 Stand-alone devices:
 
 - `Aerobits idME+ <https://www.aerobits.pl/product/idme-remoteid/>`__
+- `AEROENTRY remote id (Japan) <https://aeroentry.co.jp/info/product/228/>`__
```

Comment:
link gives me a 404 not found

Reply:
@Hwurzburg san.  I converted it to an HTML file in PANDOC and successfully displayed the page.